gittrigger/gittrigger.go

```diff
@@ -35,12 +35,36 @@ func (g *gitTrigger) getLatestCommit() error {
 	return nil
 }
 
-func (g gitTrigger) checkTrigger() bool {
+func (g *gitTrigger) GetRemote() (string, error) {
+	var out bytes.Buffer
+	cmd := exec.Command("git", "remote")
+	cmd.Stdout = &out
+	err := cmd.Run()
+
+	if err != nil {
+		return "", err
+	}
+	remote_name := strings.TrimSpace(out.String())
+
+	out.Reset()
+	cmd.Stdout = &out
+
+	cmd = exec.Command("git", "remote", "get-url", "--all", remote_name)
```

Comment:
I think this is wrong - I'm specifically asking for a single remote name; and you can't have multiple with the same name.

Reply:
The following is the situation I believe the code can't handle:
```
git init bar
cd bar
git remote add foo https://example-foo.com
git remote add moo https://example-moo.com
git remote get-url --all $(git remote)
```

This fails with:
```
usage: git remote get-url [--push] [--all] <name>

    --push                query push URLs rather than fetch URLs
    --all                 return all URLs
```
because `git remote` has multiple lines and my understanding of the `Trimspace` call on L47 is that it remotes leading + trailing spaces.

Reply:
@jbarber ok, good catch - I was trying to add two remotes with the same name, not two remotes. Will fix.

Reply:
@jbarber fixed by 27bfb66ac3bc2aa7b9725b97e44658b1bab93e0c